modules/migration/src/main/java/com/sap/xsk/migration/neo/sdk/command/tunnel/TunnelSdkCommandModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2019-2021 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.sap.xsk.migration.neo.sdk.command.tunnel;
+
+import com.google.inject.TypeLiteral;
+import com.sap.xsk.migration.neo.sdk.command.SdkCommand;
+import org.eclipse.dirigible.commons.api.module.AbstractDirigibleModule;
+
+public class TunnelSdkCommandModule extends AbstractDirigibleModule {
+
+  @Override
+  protected void configure() {
+    bind(new TypeLiteral<SdkCommand<CloseDatabaseTunnelSdkCommandArgs, CloseDatabaseTunnelSdkCommandRes>>() {
+    }).to(CloseDatabaseTunnelSdkCommand.class);
+
+    bind(new TypeLiteral<SdkCommand<OpenDatabaseTunnelSdkCommandArgs, OpenDatabaseTunnelSdkCommandRes>>() {
+    }).to(OpenDatabaseTunnelSdkCommand.class);
```

Comment:
Could be extracted into the MigrationModule class

Reply:
Yes, I've moved the bindings into the MigrationModule